schemas/bioscan-pool-xp-tube-to-traction/1-bioscan-pool-xp-tube-to-traction.avsc

```diff
@@ -0,0 +1,126 @@
+                    },
+                    {
+                        "name": "boxBarcode",
+                        "type": "string",
+                        "doc": "The template prep kit box barcode."
+                    },
+                    {
+                        "name": "insertSize",
+                        "type": [
+                            "null",
+                            "int"
+                        ],
+                        "default": null,
+                        "doc": "(Optional) The library insert size."
+                    }
+                ]
+            }
+        },
+        {
+            "name": "request",
+            "doc": "Properties of the request associated with the tube.",
+            "type": {
+                "name": "Request",
+                "type": "record",
+                "doc": "Request properties.",
+                "fields": [
+                    {
+                        "name": "costCode",
+                        "type": "string",
+                        "doc": "The cost code for billing purposes."
+                    },
+                    {
+                        "name": "genomeSize",
+                        "type": [
+                            "null",
+                            "string"
+                        ],
+                        "default": null,
+                        "doc": "(Optional) The size of the genome, to be recorded as estimate_of_gb_required."
+                    },
+                    {
+                        "name": "libraryType",
+                        "type": "string",
+                        "doc": "The type of library for the request. Must match one of the options in Traction."
+                    },
+                    {
+                        "name": "studyUuid",
+                        "type": "version4MessageUuid",
+                        "doc": "The UTF-8 byte encoded unique study UUID the request belongs to."
+                    }
+                ]
+            }
+        },
+        {
+            "name": "sample",
+            "doc": "Properties of the sample contained in the tube.",
+            "type": {
+                "name": "Sample",
+                "type": "record",
+                "doc": "Sample properties.",
+                "fields": [
+                    {
+                        "name": "sampleName",
+                        "type": "string",
+                        "doc": "The name of the sample - displayed in Traction and filterable so might be used to store an identifier."
+                    },
+                    {
+                        "name": "sampleUuid",
+                        "type": "version4MessageUuid",
+                        "doc": "The UTF-8 byte encoded unique sample UUID the request belongs to. Will be stored as external_id in Traction."
+                    },
+                    {
+                        "name": "speciesName",
```

Comment:
This will not be fillable for Bioscan tubes, we don't know what species the 9216 samples will be till they are sequenced. I guess this is the compound sample and we'll put 'mixed' or 'unknown'?

Reply:
Yes Steve said we would use "mixed".  These were his notes about these fields:

For request:
- library_type is required. Not sure what it should be.
- data_type is null for Pacbio and not required.
- estimate_of_gb_required and number of smrt_cells are not required and probably not needed
- Cost code is required and there will be one for Bioscan.
- External study id will be the uuid for the Bioscan project.

For sample:
- Sample name is required. We should probably use this field as some sort of identifier.
- External id is required. This one might be tricky as we usually have a corresponding sample in the warehouse so this will require some thought.
- Species is required. I presume in this case it will be mixed.
- Public name. I am not sure about. We can think about this one. Might be we choose an arbitrary value.
- I suspect priority_level will always be medium
- I think we can set country_of_origin to United Kingdom.